packages/extension-api-explorer/src/ExtensionApiExplorer.tsx

```diff
@@ -25,19 +25,21 @@
  */
 
 import React, { FC, useContext, useEffect, useState } from 'react'
-import ApiExplorer, {
-  SpecItems,
-  SpecItem,
-} from '@looker/api-explorer/src/ApiExplorer'
 import { IStorageValue, RunItProvider, RunItConfigurator } from '@looker/run-it'
 import { useRouteMatch } from 'react-router-dom'
 import {
   ExtensionContext,
   ExtensionContextData,
 } from '@looker/extension-sdk-react'
-import { ApiModel, upgradeSpecObject } from '@looker/sdk-codegen'
+import {
+  ApiModel,
+  getSpecsFromVersions,
+  SpecItem,
+  SpecList,
+  upgradeSpecObject,
+} from '@looker/sdk-codegen'
 import { Looker31SDK, Looker40SDK } from '@looker/sdk'
-import { getSpecsFromVersions } from '@looker/api-explorer/src/reducers'
+import ApiExplorer from '@looker/api-explorer/lib/ApiExplorer'
```

Comment:
NOTE TO SELF: Need to figure out why we have to reach in to get ApiExplorer. Unrelated to this change.